monitoring/user.env

```diff
@@ -49,6 +40,9 @@
 # Version of the kube-prometheus-stack helm chart to use
 # KUBE_PROM_STACK_CHART_VERSION=11.0.0
 
+# Initial password of the Grafana admin user
+# GRAFANA_ADMIN_PASSWORD=yourPasswordHere
+
```

Comment:
Is having a password in plaintext in their user.env file a security concern? If the user.env file contains GRAFANA_ADMIN_PASSWORD= with no value, would it randomly generate one?

Reply:
Env vars can be set in multiple ways. I thought the user.env was our only real doc on what could be set, so I put it there. In practice, it's up to the customer whether to put the plain text password in a file or source it into their environment manually prior to deployment.

Reply:
I wonder whether we should include a comment here (and in the corresponding logging file) indicating that the environment variables tied to password can also be set via other mechanism (e.g. issuing set/export command prior to running the scripts) if users need/want to avoid having passwords set in files.  And/or, include it in the doc.  Should probably also point out that setting it elsewhere means they need to remember it or take other steps to ensure it can be retrieved later.

Reply:
It's a general principle. Content of user.env files ends up as environment variables. Those are settable externally, like any environment variable. They could even set something like their KUBECONFIG in the user.env to tie a given USER_DIR to a particular cluster.

I would include this as part of the overall rework of user config documentation being handled separately.